CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 ### Added
 
 - Sarif output format now includes `fixes` section
+- Patterns will now match against code with extra parentheses. See Ruby patterns change
+  below for an example
+
+### Changed
+
+- Ruby patterns will now match arguments with extra parentheses around arguments, or with a space
```

Comment:
Nit: I think it would be more useful to give the example of `method("...")` matching `method ("...")`. I imagine I would see this and think "Why would I ever write `method(("foo"))`?"